src/GovUk.Education.ExploreEducationStatistics.Common/GovUk.Education.ExploreEducationStatistics.Common.csproj

```diff
@@ -30,7 +30,7 @@
         <PackageReference Include="Microsoft.Azure.Storage.Queue" Version="11.2.3" />
         <PackageReference Include="System.Linq.Async" Version="5.1.0" />
         <PackageReference Include="System.Net.NameResolution" Version="4.3.0" />
-        <PackageReference Include="Mime" Version="3.1.0" />
+        <PackageReference Include="Mime" Version="3.5.1" />
```

Comment:
🔥 so good to see these changes get made! Just leaving a small comment about this. Might be worth deploying this to an environment before merging if possible. I believe we had problems in the past when trying to upgrade Mime due to mismatches between the Windows OS and the version of `Mime` that the project uses. This might turn out to be nothing though :)

Reply:
Thanks for the heads up! Will probably just deploy it to dev and just see what happens, can always just revert it!